CrossrefPlugin.php

```diff
@@ -411,4 +416,93 @@ public function getAllowedDoiTypes(): array
     {
         return [Repo::doi()::TYPE_PUBLICATION, Repo::doi()::TYPE_ISSUE];
     }
+
+    /**
+     * Make additional validation checks against publishing requirements
+     *
+     * @param $hookName string
+     * @param $args array [
+     * @option array Validation errors already identified
+     * @option Publication The publication to validate
+     * @option Submission The submission of the publication being validated
+     * @option array The locales accepted for this object
+     * @option string The primary locale for this object
+     * ]
+     * @throws Exception
+     * @see PKPPublicationService::validatePublish()
+     */
+    public function validate(string $hookName, array $args): void
```

Comment:
Hook registrant functions should have a boolean return value, and should return `Hook::CONTINUE` (almost always) or `Hook::ABORT`. See the "Hook return conventions" section of https://github.com/pkp/pkp-lib/discussions/8089.